packages/web/pages/pool/[id].tsx

```diff
@@ -289,6 +290,16 @@ const Pool: FunctionComponent = observer(() => {
   const levelCta = bondLiquidityConfig?.calculateBondLevel(bondableDurations);
   const level2Disabled = bondableDurations.length === 0;
 
+  const highestAPRDailyPeriodicRate =
+    bondableDurations[bondableDurations?.length - 1]?.aggregateApr
+      .quo(new Dec(365)) // get daily periodic rate
```

Comment:
To be more correct here as well, we should try to pull hardcoded parameters from the chain data instead of making assumptions. In the future, epochs may not necessarily be every day, it could change to every week.

See this code for how to get num epochs per year

https://github.com/osmosis-labs/osmosis-frontend/blob/ae9dcaaafa4040e29f6962b124e70f412bb40ccc/packages/stores/src/queries/pool-incentives/incentivized-pools.ts#L307-L315

Reply:
Disregard, since APRs are annualized